<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game2</class>
 <widget class="QWidget" name="Game2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Taiko2</string>
  </property>
  <widget class="QLabel" name="drum_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="scoreTitle">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>211</y>
     <width>96</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="bglayer">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>511</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="picture/picture.qrc">:/bglayer.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="timeTitle">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>256</y>
     <width>96</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="comboballoon">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>178</width>
     <height>72</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="time">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>256</y>
     <width>96</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="drum_8">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>384</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="picture/picture.qrc">:/background2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="drum_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="drum_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="drum_7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mtaiko_left">
   <property name="geometry">
    <rect>
     <x>209</x>
     <y>185</y>
     <width>47</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="speed_up">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>290</y>
     <width>404</width>
     <height>89</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="explosion">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>101</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="target">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="picture/picture.qrc">:/target.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="judgement">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>90</y>
     <width>49</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="combo_100">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>50</width>
     <height>76</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="drum_0">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="countDown">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>50</width>
     <height>76</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mtaiko_right">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>185</y>
     <width>47</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="playerchar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-34</y>
     <width>180</width>
     <height>160</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="score">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>211</y>
     <width>96</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="mtaiko">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>185</y>
     <width>512</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="picture/picture.qrc">:/mtaiko.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="restart">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>350</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
  </widget>
  <widget class="QLabel" name="drum_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="combo_10">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>50</width>
     <height>76</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="combo_1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>50</width>
     <height>76</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="drum_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="drum_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="drum_9">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>speed_up</zorder>
  <zorder>countDown</zorder>
  <zorder>playerchar</zorder>
  <zorder>mtaiko</zorder>
  <zorder>restart</zorder>
  <zorder>comboballoon</zorder>
  <zorder>combo_10</zorder>
  <zorder>combo_1</zorder>
  <zorder>combo_100</zorder>
  <zorder>score</zorder>
  <zorder>timeTitle</zorder>
  <zorder>scoreTitle</zorder>
  <zorder>time</zorder>
  <zorder>mtaiko_right</zorder>
  <zorder>mtaiko_left</zorder>
  <zorder>bglayer</zorder>
  <zorder>target</zorder>
  <zorder>explosion</zorder>
  <zorder>judgement</zorder>
  <zorder>drum_1</zorder>
  <zorder>drum_3</zorder>
  <zorder>drum_6</zorder>
  <zorder>drum_7</zorder>
  <zorder>drum_8</zorder>
  <zorder>drum_9</zorder>
  <zorder>drum_5</zorder>
  <zorder>drum_2</zorder>
  <zorder>drum_4</zorder>
  <zorder>drum_0</zorder>
 </widget>
 <resources>
  <include location="picture/picture.qrc"/>
 </resources>
 <connections/>
</ui>
